Server/controllers/auth.controller.js

```diff
@@ -1,13 +1,57 @@
 "use strict";
+const passport = require('passport');
+var jwt = require('jsonwebtoken');
+const passportConfig = require('../config/passport.config');
 /**
  * Class representing auth controller
  *
  * @class authController
  */
 class authController {
 
-    static login (req, res) {}
+  static login (req, res , next) {
+      passport.authenticate('local', function(err, user, info) {
+          if(err) {
+              return next(err);
+          }
+      
+          if (!user) { return res.json( { message: info.message }) }
```

Comment:
Please correct code indentation etc, like if condition and function statement should go in diff lines

Reply:
Done